### PR TITLE
Two Twelve: Support building on 2.12.

### PR DESCRIPTION
**Pull request by Shadowfiend**
For your consideration, @farmdawgnation .

### CODE REVIEW

**Review by farmdawgnation — APPROVED** [review]
Pushed one minor change to actually add 2.12 to the build definition. Other than that, it's looking good! :)

**Comment by Shadowfiend** [comment]
Cool. I need to figure out the 2.6 lift cross-build too <_<

**Comment by Shadowfiend** [comment]
So I actually think this is good to go… Would be nice to add snapshot publishing on merge to master though…

**Comment by Shadowfiend** [comment]
Ok, pending some travis love, we should be in business here…

We now cross-build against Scala 2.11 and 2.12 for Lift versions 3.0, 3.1, and 3.2; we also build against Scala 2.11 for Lift version 2.6! The patch versions are the latest across the major/minor versions.

We also auto-publish on the master branch for maximum joy. I say that, but we'll see if it works in practice once the merge happens………

File: .travis.yml [hunk 1]
```diff
@@ -3,11 +3,25 @@ jdk:
...
+cache:
+  directories:
+    - $HOME/.ivy2
+
+env:
+  - LIFT_VERSION=3.2.0
+  - LIFT_VERSION=3.1.1
+  - LIFT_VERSION=3.0.2
+
+# Add a spritz of 2.6 build.
```
Comment (farmdawgnation) [inline .travis.yml:18]: just a spritz

File: travis.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+if grep -q -e "version :=.*SNAPSHOT" build.sbt && [ "${TRAVIS_PULL_REQUEST}" == "false" ] && [ "${TRAVIS_BRANCH}" == "master" ] ; then
+  echo "Publishing snapshot to sonatype..."
+
+  openssl aes-256-cbc -K $encrypted_ddfe6500c86a_key -iv $encrypted_ddfe6500c86a_iv -in .sonatype.enc -out ~/.sonatype -d
+
+  sbt ++$TRAVIS_SCALA_VERSION "set liftVersion := \"$LIFT_VERSION\"" publish
+
+  rm ~/.sonatype
+fi
```
Comment (farmdawgnation) [inline travis.sh]: Missing newline to end the file

**Review by farmdawgnation — APPROVED** [review]
Picked a nit, but otherwise good to go. :+1:

**Comment by Shadowfiend** [comment]
Fix't, merging on green light!

**Pull request merged**